http/vibe/http/server.d

```diff
@@ -1012,7 +1012,6 @@ final class HTTPServerRequest : HTTPRequest {
 
 		private void parseFormAndFiles() @safe {
 			_form = FormFields.init;
-			assert(!!bodyReader);
```

Comment:
This is moved to `parseFormData`, because if the content type is not a form, `bodyReader` isn't used.

Makes unit testing much easier, since I don't have to invent a null input stream (which is suspiciously missing).